templates/ohdsi-webapi/keyvault.bicep

```diff
@@ -0,0 +1,87 @@
+param location string
+param odhsiWebApiName string
+param tenantId string
+param suffix string
+@secure()
+param databaseAdminPassword string
```

Comment:
consider renaming to `postreSQLAdminPassword` which implies the top level admin user.

Reply:
done